content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md

```diff
@@ -304,7 +304,7 @@ spec:
     resource:
       name: cpu
       target:
-        type: AverageUtilization
+        type: Utilization
```

Comment:
According to https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#metrictarget-v2beta2-autoscaling,
The type should be one of Utilization, Value, or AverageValue.
So this change makes sense to me.